src/coffee/client/modules/client_schema.coffee

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (C) 2015 by Looker Data Services, Inc.
+# All rights reserved.
+#
+
+angular = require 'angular'
+
+############################################################################################################
+
+schema = angular.module 'schema', ['js-data']
+
+schema.run (DS, DSLocalStorageAdapter) ->
+  DS.registerAdapter('localStorage', DSLocalStorageAdapter, { default: true })
+
+schema.factory 'Todo', (DS) -> DS.defineResource require '../../model/todo_model'
+
+# force the services to be created so that recursive lookups succeed
+schema.run (Todo) -> # do nothing
```

Comment:
Here's an example of an "odd" piece of code I documented so that someone doesn't  "clean it up" later.